src/image.rs

```diff
@@ -412,6 +412,11 @@ pub trait ImageDecoder<'a>: Sized {
     fn into_reader(self) -> ImageResult<Self::Reader>;
 
     /// Returns the total number of bytes in the decoded image.
+    ///
+    /// This is the size of the buffer that must be passed to `read_image` or
+    /// `read_image_with_progress`. It is possible that the returned value exceeds usize::MAX, in
```

Comment:
It may verbose a bit.
```suggestion
    /// `read_image_with_progress`. The returned value may exceed usize::MAX, in
```